### PR TITLE
Add slack 2

### PR DESCRIPTION
**Pull request by aarondr77**
# Description

Moves most of the Discord links to Slack links. 

# Testing

Make sure the Slack link works + adds you to the channels general, install-help, python-support, feature-ideas. Make sure you cannot see any private channels. 

# Documentation

Once this is tested, I'll update the docs too.

### CODE REVIEW

File: mitosheet/src/components/elements/ErrorBoundary.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ class ErrorBoundary extends Component<Props, State> {
         if (this.state.hasError) {
             return (
                 <p className='text-body-1 text-color-red p-10px'>
-                    Looks like Mito had an error! Sorry about that. Rerun the Jupyter Cell above, and <a className='text-body-1-link' href={DISCORD_INVITE_LINK} target='_blank' rel="noreferrer">join our Discord for support</a> if this error occurs again.
+                    Looks like Mito had an error! Sorry about that. Rerun the Jupyter Cell above, and <a className='text-body-1-link' href={SLACK_INVITE_LINK} target='_blank' rel="noreferrer">join our Slack for support</a> if this error occurs again.
```
Comment (naterush) [inline mitosheet/src/components/elements/ErrorBoundary.tsx:36]: Can we keep discord and slack so users have options here?

File: mitosheet/src/components/modals/ErrorModal.tsx [hunk 2]
```diff
@@ -65,7 +65,7 @@ const ErrorModal = (
                     <TextButton
                         variant='dark'
                         width='medium'
-                        href={DISCORD_INVITE_LINK}
+                        href={SLACK_INVITE_LINK}
```
Comment (naterush) [inline mitosheet/src/components/modals/ErrorModal.tsx:68]: I'd be down to randomize between the two of them, and see what people click more!

File: trymito.io/components/Footer/Footer.tsx [hunk 3]
```diff
@@ -78,7 +81,7 @@ const Footer = (): JSX.Element => {
                         <a href='https://www.youtube.com/channel/UCN9o_0m1fwCjigfIpnKr0oA/videos' target="_blank" rel="noreferrer">YouTube</a>
                     </li>
                     <li className='text-nav'>
-                        <a href='https://discord.gg/XdJSZyejJU' target="_blank" rel="noreferrer">Contact</a>
+                        <a href='[Slack](https://join.slack.com/t/trymito/shared_invite/zt-1h6t163v7-xLPudO7pjQNKccXz7h7GSg)' target="_blank" rel="noreferrer">Contact</a>
```
Comment (naterush) [inline trymito.io/components/Footer/Footer.tsx:84]: I feel like this would be a good spot to have both of them for sure!

**Review by naterush — CHANGES_REQUESTED** [review]
Can we add both discord and slack where it is appropriate?

File: mitosheet/src/components/modals/ErrorModal.tsx (hunk 2 above)
Comment (aarondr77) [reply mitosheet/src/components/modals/ErrorModal.tsx:68]: For now, I think the most important thing is just engagement. Discord has almost 0 engagement, so let's see if people are on Slack more and will post more. This button also just says "Get immediate support" anyways.

File: trymito.io/components/Footer/Footer.tsx (hunk 3 above)
Comment (aarondr77) [reply trymito.io/components/Footer/Footer.tsx:84]: Its already in the sitemap. There is a button for Discord and Slack.

**Pull request merged**